persist/fs/clone/cloner.go

```diff
@@ -31,15 +31,28 @@ func (c *cloner) Clone(src FilesetID, dest FilesetID, destBlocksize time.Duratio
 	if err != nil {
 		return fmt.Errorf("unable to create fileset reader: %v", err)
 	}
-	if err := reader.Open(ident.StringID(src.Namespace), src.Shard, src.Blockstart); err != nil {
+	openOpts := fs.ReaderOpenOptions{
+		Namespace:  ident.StringID(src.Namespace),
+		Shard:      src.Shard,
+		BlockStart: src.Blockstart,
+		IsSnapshot: false,
```

Comment:
Can we make this an enum rather than a bool `IsSnapshot` in case have other types of fileset files?

Reply:
I can, although its gonna clutter up the code some because I'm gonna have to convert if/else statements to switch statements with defaults that return customer errors etc etc

I think it made a lot of sense to future-proof the file-format on disk, but making the change in the code-base itself is easy enough that it might be worth leaving as is until we need to change it

Reply:
done